omod-2.0/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_0/ConceptAttributeTypeController2_0Test.java

```diff
@@ -73,4 +74,19 @@ public void shouldCreateConceptAttributeType() throws Exception {
 
     }
 
+    @Test
+    public void shouldUpdateConceptAttributeType() throws Exception {
+        ConceptAttributeType existingConceptAttributeType = service.getConceptAttributeTypeByUuid("858472ac-c220-4c7f-9990-980c176c6099");
```

Comment:
If we changed this uuid, in how many places would we do it?

Reply:
I mean in this same file.

Reply:
two places @dkayiwa bt in this new changes have done so

Reply:
Why should you change the same value in more than one place?

Reply:
My reason is since it is a new object...and because already the other object is existing yet this is updated otherwise any Suggestion to do better

Reply:
@dkayiwa can I assign it a Variable

Reply:
https://www.techrepublic.com/article/avoid-using-magic-numbers-and-string-literals-in-your-code/

Reply:
Got the thing @dkayiwa 🔤